modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -1711,7 +1689,7 @@ func (suite *KeeperTestSuite) TestAbortHandshake() {
 
 				channel, found := channelKeeper.GetChannel(suite.chainA.GetContext(), path.EndpointA.ChannelConfig.PortID, path.EndpointA.ChannelID)
 				if found { // test cases uses a channel that exists
-					suite.Require().Equal(types.INITUPGRADE, channel.State, "channel state should not be restored to %s", types.INITUPGRADE.String())
+					suite.Require().Equal(types.OPEN, channel.State, "channel state should not be restored to %s", types.OPEN.String())
```

Comment:
ditto for `.String` and ditto for punting this in the future

Reply:
ah ok https://github.com/cosmos/ibc-go/issues/3946

Reply:
aye, it seems there's a bit of a mix and match free for all at the moment, we can standardise these later